posthog/management/commands/create_ch_migration.py

```diff
@@ -31,8 +31,8 @@ def handle(self, *args, **options):
         idx = len(entries)
         index_label = _format_number(idx)
         file_name = "{}/{}_{}".format(MIGRATION_PATH, index_label, name)
-        f = open(file_name, "w")
-        f.write(FILE_DEFAULT)
+        with open(file_name, "w", encoding="utf_8") as f:
```

Comment:
Should this be `utf-8` rather than `utf_8`? I've never seen it written with an underscore in Python before

Reply:
We can change it if you like but the [official codec](https://docs.python.org/3/library/codecs.html#standard-encodings) is `utf_8` (that has aliases like U8, UTF, utf8, cp65001).

On top of that:
> Notice that spelling alternatives that only differ in case or use a hyphen instead of an underscore are also valid aliases; therefore, e.g. 'utf-8' is a valid alias for the 'utf_8' codec.

Reply:
Nope I'm happy! TIL, thank you 🙌